src/test/java/project/bookstore/controller/util/BookUtilTest.java

```diff
@@ -0,0 +1,103 @@
+package project.bookstore.controller.util;
+
+import java.math.BigDecimal;
+import java.util.List;
+import org.springframework.stereotype.Component;
+import project.bookstore.dto.book.BookDto;
+import project.bookstore.dto.book.CreateBookRequestDto;
+
+@Component
+public class BookUtilTest {
+    public CreateBookRequestDto createBookRequestDto() {
```

Comment:
for all other methods
```suggestion
    public static CreateBookRequestDto createBookRequestDto() {
```